WalletWasabi/Blockchain/TransactionBroadcasting/TransactionBroadcaster.cs

```diff
@@ -22,24 +22,24 @@ public class TransactionBroadcaster
 {
 	public TransactionBroadcaster(Network network, BitcoinStore bitcoinStore, WasabiHttpClientFactory httpClientFactory, WalletManager walletManager)
 	{
-		Network = Guard.NotNull(nameof(network), network);
-		BitcoinStore = Guard.NotNull(nameof(bitcoinStore), bitcoinStore);
+		Network = network;
+		BitcoinStore = bitcoinStore;
 		HttpClientFactory = httpClientFactory;
-		WalletManager = Guard.NotNull(nameof(walletManager), walletManager);
+		WalletManager = walletManager;
 	}
 
-	public BitcoinStore BitcoinStore { get; }
-	public IWasabiHttpClientFactory HttpClientFactory { get; }
-	public Network Network { get; }
-	public NodesGroup? Nodes { get; private set; }
-	public IRPCClient? RpcClient { get; private set; }
-	public WalletManager WalletManager { get; }
+	private BitcoinStore BitcoinStore { get; }
+	private IWasabiHttpClientFactory HttpClientFactory { get; }
+	private Network Network { get; }
+	private NodesGroup? Nodes { get; set; }
+	private IRPCClient? RpcClient { get; set; }
+	private WalletManager WalletManager { get; }
 	private WasabiRandom Random { get; } = InsecureRandom.Instance;
 
-	public void Initialize(NodesGroup nodes, IRPCClient? rpc)
+	public void Initialize(NodesGroup nodes, IRPCClient? rpcClient)
 	{
 		Nodes = Guard.NotNull(nameof(nodes), nodes);
```

Comment:
If we are removing `Guard.NotNull`s from this class, maybe we can remove it from LOC41, too.

```suggestion
		Nodes = nodes;
```